Public/Src/Cache/ContentStore/App/Hello.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using BuildXL.Cache.ContentStore.Exceptions;
+using BuildXL.Cache.ContentStore.Hashing;
+using BuildXL.Cache.ContentStore.Interfaces.FileSystem;
+using BuildXL.Cache.ContentStore.Interfaces.Tracing;
+using BuildXL.Cache.ContentStore.Service;
+using BuildXL.Cache.ContentStore.Service.Grpc;
+using Microsoft.Practices.TransientFaultHandling;
+using CLAP;
+using BuildXL.Cache.ContentStore.Sessions;
+using Grpc.Core;
+using ContentStore.Grpc;
+using System.Linq;
+
+namespace BuildXL.Cache.ContentStore.App
+{
+    internal sealed partial class Application
+    {
+        public static ChannelOption[] DefaultChannelOptions = new ChannelOption[] { new ChannelOption(ChannelOptions.MaxSendMessageLength, -1), new ChannelOption(ChannelOptions.MaxReceiveMessageLength, -1) };
+
+        /// <summary>
+        /// Attempt to connect to a grpc port and send 'Hello'
+        /// </summary>
+        [SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
+        [Verb(Description = "Send 'Hello' to another CASaaS")]
+        internal void Hello(

```

Comment:
>Hello [](start = 22, length = 5)

Are we sure we want to call this Hello? Sounds very old-school... `Ping` , `Initialzie`, `IsAlive`, `IsActive` seems more appropriate?

Reply:
I think this is done for consistency.

---
In reply to: [274110409](https://github.com/Microsoft/BuildXL/pull/120#discussion_r274110409) [](ancestors = 274110409)

Reply:
Correct, this is for consistency. 

---
In reply to: [274115371](https://github.com/Microsoft/BuildXL/pull/120#discussion_r274115371) [](ancestors = 274115371,274110409)